objc/src/main/java/org/robovm/objc/ObjCObject.java

```diff
@@ -455,25 +507,25 @@ static void release(@Pointer long self, @Pointer long sel) {
             }
             ObjCRuntime.void_objc_msgSendSuper(new Super(self, NS_OBJECT_CLASS).getHandle(), sel);
```

Comment:
Yes that immediately crashes the app. I think the first argument of class_getSuperclass needs to be a class handle not an object handle. But when I pass a class handle via object_getcClass, I get a stackoverflow error.